scss/bitstyles/utilities/break-text/_index.scss

```diff
@@ -1,6 +1,17 @@
 @use '../../settings/setup';
 @use '../../tools/break-text';
+@use '../../tools/properties';
 
-.#{setup.$namespace}u-break-text {
+/* stylelint-disable scss/dollar-variable-default */
```

Comment:
I’m not sure what you mean by `local` — Sass has no local keyword (variables are scoped to the block they’re declared in, which could be global if declared at root, or local when declared inside a `{}` bracketed block.

These classname variables are only there to make the code readable, and are not overridable by users of bitstyles  — variables need to be declared with `!default` to make them overridable. That’s why this linting rule is default enabled, since we normally want all our variables to be overridable.